params/config.go

```diff
@@ -347,72 +332,89 @@ type XDPoSConfig struct {
 }
 
 type V2 struct {
-	FirstSwitchBlock *big.Int             `json:"switchBlock"`
+	SwitchBlock      *big.Int             `json:"SwitchRound"`
 	CurrentConfig    *V2Config            `json:"config"`
 	AllConfigs       map[uint64]*V2Config `json:"allConfigs"`
 	configIndex      []uint64             //list of switch block of configs
 	SkipV2Validation bool                 //Skip Block Validation for testing purpose, V2 consensus only
 }
 
 type V2Config struct {
+	SwitchRound          *big.Int `json:"switchRound"`          // v1 to v2 switch block number
```

Comment:
use types.Round or uint64

Reply:
+1, it can only use uint64, `types.Round` will issue the cycle import